mthamayor/auto-mart
# #166284574 Admins can delete cars

#### What does this PR do?
Admin can delete car adverts

#### Description of Task to be completed?
Create API endpoint DELETE /car/<:car_id>/
Unit tests should be written for the endpoint
The endpoint should be tested on Postman

#### How should this be manually tested?
After cloning the repo

Start the development server with
npm run dev
After signing in, make sure youre an admin and there is an existing purchase advert
Get the id of the car
On postman, set the Authorization header with the value
Bearer <token>
the URL is
http://localhost:3000/api/v1/car/<car_id>/
Hit send on postman
You should receive a response similar to 
```
{
    "status": 200,
    "data": "Car Ad successfully deleted"
}
```
#### Any background context you want to provide?
You would need to be an admin before you can delete any post

#### What are the relevant pivotal tracker stories?
https://www.pivotaltracker.com/story/show/166284574
